src/lando/main/models/landing_job.py

```diff
@@ -125,6 +156,26 @@ def landed_revisions(self) -> dict:
         )
         return dict(revision_landing_jobs)
 
+    @property
+    def landed_treeherder_revision(self) -> str | None:
+        """Return a revision suitable for use with TreeStatus."""
+        if not self.landed_commit_id:
+            return None
+
+        th_revision = None
+        if self.target_repo.scm_type == SCM_TYPE_HG:
+            th_revision = self.landed_commit_id
+        elif self.target_repo.scm_type == SCM_TYPE_GIT:
+            try:
+                th_revision = CommitMap.git2hg(
+                    self.target_repo.name, self.landed_commit_id
+                )
+            except CommitMap.DoesNotExist:
+                logger.warning(
+                    f"CommitMap not found for {self.landed_commit_id} in {self.target_repo.name}"
+                )
+        return th_revision
```

Comment:
Nit: you could avoid setting a default `None` by returning everywhere you set `th_revision`. That would allow you to avoid the `elif` block, which would de-dent the `try/except`.

Reply:
Ah, yeah, that's nicer.